### PR TITLE
TFSS-23 Subject generation directory

### PR DESCRIPTION
**Pull request by PierreLessard**
Basic file structuring for ticket description generation. Includes gitignored config.py that requires chat GPT API key variable.

### CODE REVIEW

File: email_processing/ticket_description_generation/description_generator.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from config import API_KEY
```
Comment (youssefsoli) [inline email_processing/ticket_description_generation/description_generator.py:1]: I recommend adding config.py and making it read from an .env file instead. Then you can have an example env file for developers to copy and edit.

File: README.md [hunk 2]
```diff
@@ -10,4 +10,4 @@ Caesar Saleh
 Youssef Soliman
 Sarah Xu
 Sataphon 
-Valerie Yip
\ No newline at end of file
+Valerie Yip
```
Comment (youssefsoli) [inline README.md:13]: This shouldn't be in this PR

**Review by youssefsoli — CHANGES_REQUESTED** [review]
Good directory structure, make sure to create a ticket on Linear and comment it in the description for tracking.

File: email_processing/ticket_description_generation/description_generator.py (hunk 1 above)
Comment (PierreLessard) [reply email_processing/ticket_description_generation/description_generator.py]: Much better idea, especially for values shared across directories, implemented

File: README.md (hunk 2 above)
Comment (PierreLessard) [reply README.md]: My mistake, branched from an older commit by mistake and thought a merge would fix it but needed to do a single file checkout instead.

**Review by youssefsoli — APPROVED** [review]
Thanks for the changes!

**Pull request merged**